### PR TITLE
Add LMY Token

### PR DESCRIPTION
**Pull request by LunchMoneyToken**
Dear General Bytes Team,

As requested, please integrate Lunch Money (LMY) Token in your platform.

Sincerely,

Lunch Money Team

### CODE REVIEW

File: server_extensions_extra/src/main/resources/lunchmoney.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...lrl4hIlLhjUE7MWHWPwYyqWowcKD+GlFLAzzikmJYfiSXiGixWrbJtDQE/kovmVIr1Y4S+x30XX81h6NWi+GSRFPBgiVLFi0UEsQGPd4aFAqB+BtM/COYtRP1HdCtQdeffJaXKA+rjEmK8wRl4xEF/xYprmzK8Ud/C41Lg0UOokazygw/NN6q+ahrq8lwIj/8Q/tnep1IyOFQOv6xzJqB+qO8vEhxcnx4XLinl7+aAuNKloTBiepaYWaZv8hJwFtXheCjwG+z8her6XEYWAsq5nC4f3Eg/lGREbSN+o1Pv/wYKXthlEd4XIynmx9bhPb5RCJiv096bW+CWuUooshLmUuD8ZHw2r4lASIqXIS2jZQaPQBzA595/stvKOIfqyM7U99OUIMPP/k1KDsxUoaxw0NZIf5skQ1QyIlVOYv4u30r2OQeKBULCgZepRLWEgDMAxUj4mehH4OOfwrESAjqX86Y+TSY+Kdff/xRImDzk6M9w8PDPVlewUGiCay2zV7axi/CZev8/ZS+vr5Kf7aQGxjIe+bvhIp/YsRICAuHkQsr+5svMbZCEhYV7RkaHhrqAeBY47EMgmjZQf5+vj4gSi07lhPI4/H/gYCxipFR//j92FkoaqxswAYj/wC45ZKw5BB3j1AQTw+Wu1eIn79/UBAbBJwU+sMOAlS/EB8vb3RLx8fPXyTkAi5f8DwJDFOumT9JYiyIGmPfg7E/Rer+IDEyUiETBwYEhbixPDyxeICwsLiD4N9uiJatAu0CrUAMCkZ9mLLpXxJG/dMjRnKbIzZwf/3lJx9/8AFwK+TofgUngB0UHBLiRUpICOg9iB0gRLACgBVL/vE85kUKpoAJFf80iZHcRvgzbOQU9zcI/KMPPlCA4Js1EolELBYDISESLP/4x/MYF+sXKfienwUwIQQ21raN++SniBzQP37++Q8++OAfdHkeWD/5EtMCLqFf4KUB/9SJsVDYFDeAY3LE/uk333zztU3gBFAJ2DcwLsmL+/DPBpiQ22jcGByRAzqCx3KSwHwDoz737LOIFuNi/QLvzwyYFIobgWPyGTOBHdHTBYXMBFhEi3F/tryUENwIHMgBHdixzMBCnvwSwSJaCvfny0vJbRQ5wU7yI8pfUOcIlqD9+ePShCC30dvkNrv82E28ZXKbi58fVv4fqy+taiD89b4AAAAASUVORK5CYII="/>
```
Comment (generalbytes) [inline server_extensions_extra/src/main/resources/lunchmoney.svg:3]: Raster image is not supported inside SVG. Use only vectors inside SVG.

**Review by generalbytes — CHANGES_REQUESTED** [review]

**Comment by generalbytes** [comment]
Raster image is not supported inside SVG. Use only vectors inside SVG.

File: server_extensions_extra/src/main/resources/lunchmoney.svg (hunk 1 above)
Comment (LunchMoneyToken) [reply server_extensions_extra/src/main/resources/lunchmoney.svg:3]: Thank you. We'll resubmit the logo in the proper format.

Comment (LunchMoneyToken) [reply server_extensions_extra/src/main/resources/lunchmoney.svg]: updated svg

**Comment by LunchMoneyToken** [comment]
@generalbytes the svg is updated.

**Pull request merged**